src/lib.rs

```diff
@@ -446,6 +448,88 @@ where
     }
 }
 
+#[cfg(not(feature = "sink"))]
+impl<S> WebSocketStream<S> {
+    /// Simple send method to replace `futures_sink::Sink` (till v0.3).
+    pub async fn send(&mut self, msg: Message) -> Result<(), WsError>
```

Comment:
Doesn't hurt to provide this always, and should make sure that both this and the `Sink` impl are covered by the tests and running on the CI

Reply:
I think apart from this one it's all ready

Reply:
The new function is already used by everything (e.g. the autobahn tests). Maybe you can change it so that the server uses `Sink::send` and the client the new function (together with a comment why this is done)